evalai/utils/requests.py

```diff
@@ -105,3 +109,16 @@ def make_request(path, method, files=None, data=None):
     elif method == "DELETE":
         # TODO: Add support for DELETE request
         pass
+
+
+def check_compatibility(response):
+    if "Minimum-CLI-Version" in response.headers:
+        version = pkg_resources.require("evalai")[0].version
+        if semver.compare(version, response.headers["Minimum-CLI-Version"]) is -1:
+            echo(
+                style(
+                    "\nYour CLI ({}) is too old and needs to be updated\n".format(version),
```

Comment:
What do you think about:
```py
"\nYour CLI is too old and needs to be updated from {0} to {1}\n".format(version, response.headers["Minimum-CLI-Version"])
```

Reply:
Sounds like a good idea to get more info

Reply:
What do you think about the following message:
```py
Current CLI is not compatible with EvalAI hosted at [SERVER_HOST]. It needs to be updated from [CLI_VERSION] to [Minimum_CLI_Version]
```